.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-  "presets": [["@verdaccio", {"typescript": true}]],
-  "plugins": ["dynamic-import-node"]
+  "presets": ["@babel/env", "@babel/typescript"],
+  "plugins": ["dynamic-import-node", "@babel/proposal-nullish-coalescing-operator", "@babel/proposal-optional-chaining"]
```

Comment:
Do we really need these plugins now?

They respective ES features are no longer proposals (their are part of the spec now) and the docs for these packages seems to redirect to the corresponding transform plugins: [@babel/plugin-transform-nullish-coalescing-operator][1] and [@babel/plugin-transform-optional-chaining][2]

Also, both plugins mention on their docs:

> **NOTE:** This plugin is included in `@babel/preset-env`, in [ES2020](https://github.com/tc39/proposals/blob/master/finished-proposals.md)

##
Can you please investigate whether we can remove them altogether (and if not switch to their `plugin-transform-*` counterparts)?

[1]: https://www.npmjs.com/package/@babel/plugin-transform-nullish-coalescing-operator
[2]: https://www.npmjs.com/package/@babel/plugin-transform-optional-chaining

Reply:
You are right! We do not need both the `proposal-*` and `plugin-transform-*` plugins. These plugins have reached stage 4 ("Finished") of the [TC39](https://tc39.es/process-document/) process and are now officially part of ES2020 ([tc39/proposal-optional-chaining](https://github.com/tc39/proposal-optional-chaining), [tc39/proposal-nullish-coalescing](https://github.com/tc39/proposal-nullish-coalescing)). As a result, the TypeScript compiler will emit both the nullish operator `??` and the optional operator `?` without any downleveling when we set the target `ES2020` (or a newer version of the language) in tsconfig. 

In this project, we [set the target](https://github.com/xlts-dev/verdaccio-google-cloud-storage/blob/master/tsconfig.json#L3) to `esnext`, thus supporting both operators by default. However, if we need to transpile the code to an earlier ECMAScript version that doesn't natively support these features, then we would require these Babel plugins.

Reply:
Removed plugins `@babel/proposal-nullish-coalescing-operator` and `@babel/proposal-optional-chaining`